examples/port-2-alibaba-connection-with-alicloud-terraform-integration/README.md

```diff
@@ -0,0 +1,50 @@
+# Fabric Port to Fabric Alibaba Profile Connection
+
+This example shows how to leverage the [Fabric Port Connection Module](https://registry.terraform.io/modules/equinix/fabric/equinix/latest/submodules/port-connection)
+to create a Fabric Connection from a Fabric Port to Fabric Alibaba Service Profile.
+
+It leverages the Equinix Terraform Provider, the Alibaba Terraform Provider, and the Fabric Port Connection
+Module to setup the connection based on the parameters you have provided to this example; or based on the pattern
+you see used in this example it will allow you to create a more specific use case for your own needs.
+
+See example usage below for details on how to use this example.
+### Step by Step Instructions for Fabric Port to Fabric Alibaba Profile Connection
+
+#### 1. Create Connection from Equinix Terraform Provider
+
+**Note:** The `connection_name` must follow either one of these patterns:
+
+* A unique `connection_name` with atmost 24 characters.  
+`connection_name = var.connection_name`
+* A `connection_name` with atmost 12 characters combined with a random string of 12 characters:  
+  `connection_name = "${var.connection_name}${random_string.random.result}"`  
+  Use the `random_string` resource to perform this operation:
+  ```hcl
+  resource "random_string" "random" {
+    length  = 12
+    special = false
+  }
+  ```
+  * A unique `connection_name` is generated by appending a 12-character random string to the variable name given by the user.
+  * A Fabric Port to Alibaba Profile Connection is successfully created and displayed in the Fabric Portal.
+  * An Express Connect Physical Connection resource is created and becomes visible in the Alibaba Portal.
+
+#### 2. Formulate main.tf and output.tf Files
+* When you run the initial `terraform apply`, the resources and data sources (from the example) are in comments.
+* The comments can be removed after you accept the connection with the Alibaba portal.
+
+#### 3. Accept the Connection Request
+* Manually accept the connection request in the Alibaba Portal for the created physical connection.
+
+#### 4. Create the Virtual Border Router (VBR) Resource
+* Remove the commented code to create the VBR resource.
```

Comment:
@thogarty  Should I add in here that they need to do these commands after removing commented code - 
`terraform init`
`terraform plan`
`terraform apply`

Reply:
Just the init and apply. They don't need to run the plan.

Reply:
We need to do `terraform init` for provider "alicloud"